src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -305,14 +311,14 @@ private CompletableFuture<Response> processSingleUser(JsonObject userJson, UserR
       logger.info("processSingleUser:: Retrieved remote patron group ID: {}", remotePatronGroupId);
 
       if (!isActive) {
-        logger.warn("processSingleUser:: User account is not active");
-        return PostPatronAccountResponse.respond422WithApplicationJson(createError("User account is not active", String.valueOf(HTTP_UNPROCESSABLE_ENTITY)));
+        logger.warn("processSingleUser:: {}", USER_ACCOUNT_INACTIVE.value());
+        return PostPatronAccountResponse.respond422WithApplicationJson(createError(USER_ACCOUNT_INACTIVE.name(), String.valueOf(HTTP_UNPROCESSABLE_ENTITY)));
       } else if (remotePatronGroupId.equals(patronGroup)) {
-        logger.warn("processSingleUser:: User already exists in the remote patron group");
-        return PostPatronAccountResponse.respond422WithApplicationJson(createError("User already exists", String.valueOf(HTTP_UNPROCESSABLE_ENTITY)));
+        logger.warn("processSingleUser:: {}", USER_ALREADY_EXIST.value());
+        return PostPatronAccountResponse.respond422WithApplicationJson(createError(USER_ALREADY_EXIST.name(), String.valueOf(HTTP_UNPROCESSABLE_ENTITY)));
       } else {
-        logger.warn("processSingleUser:: User does not belong to the required patron group");
-        return PostPatronAccountResponse.respond422WithApplicationJson(createError("User does not belong to the required patron group", String.valueOf(HTTP_UNPROCESSABLE_ENTITY)));
+        logger.warn("processSingleUser:: {}", INVALID_PATRON_GROUP.value());
+        return PostPatronAccountResponse.respond422WithApplicationJson(createError(INVALID_PATRON_GROUP.name(), String.valueOf(HTTP_UNPROCESSABLE_ENTITY)));
```

Comment:
As String.valueOf(HTTP_UNPROCESSABLE_ENTITY) will always return a constant value, I think we can use constant for this. what do you think?

Reply:
I think this is itself is constant, it will just call toString() from enum which give enum constant only, I don't think CPU cycle will reduce by putting this in constant. I think this would be over if we add one more constant just to do toString().

For maintainability of code as well this would be a over by putting this enum constant again in class level constant field. In future if more statuses came from HttpStatus then this class(PatronServicesResourceImpl) itself will filled with enum constants.

So I think its okay and we can keep this way only.
Thanks.